unitTests/testArrayOfSets.cpp

```diff
@@ -890,6 +828,47 @@ class ArrayOfSetsViewTest : public ArrayOfSetsTest< typename ARRAY_OF_SETS_POLIC
     COMPARE_TO_REFERENCE
   }
 
+  // This is not a View test, but it needs a POLICY type argument, so it's placed here for convenience
+  void assimilate( IndexType const maxValue, IndexType const maxInserts, sortedArrayManipulation::Description const desc )
```

Comment:
Lmk if you'd like a separate test sub-class to contain these assimilation tests (I moved them from `ArrayOfSetsTest` to `ArrayOfSetsViewTest` for reason stated in the code comment).

Reply:
Nope that sounds just fine. I've done similar things elsewhere.